controllers/message.js

```diff
@@ -13,9 +14,14 @@ const sendMessage = async (req, res) => {
 const getMessages = async (req, res) => {
   const { id: user } = req.user
   const { id: chat } = req.params
-  const { skip, limit } = req.query
+  const { skip, limit, message } = req.query
+  const searchCondition = { chat }
 
-  const messages = await messageService.getMessages({ user, chat }, parseInt(skip), parseInt(limit))
+  if (message) {
```

Comment:
i belive using getMatchOptions (u should pass message and chat as a params) would be better here

Reply:
In this way, it will find messages only if the texts completely match

Reply:
yes i was testing getMatchOptions, in that way we can only search for complete text field.